CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# thumbnails4j Contributor's Guide
+
+Thank you for your interest in contributing to thumbnails4j!
+
+How to build and contribute to thumbnails4j.
+
+### Requirements
+
+- Java 8
+- OS: Unix/Linux
+
+### Installing dependencies
+
+From the root level of this repository:
+
+```shell
+./mvnw clean install -DskipTests
+```
+
+### Building
+
+For all projects, run from project root. For single module, run from
+module root.
+
+```shell
+# Build
+./mvnw clean install -DskipTests
```

Comment:
This is the same shell command as the one before, but I bet that's intentional?

Reply:
Yeah, there's really no magic here. I just copied the structure from the search-ui docs, which is more complex. But since it might be more complex one day, I figured it couldn't hurt to leave the multiple (redundant) sections.